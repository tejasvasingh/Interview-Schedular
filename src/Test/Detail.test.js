import React from 'react';
import { shallow } from 'enzyme';
import Detailcomp from '../Test/Comp/Detailcomp';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

it('creating details compoennet snapshot', () => {
 
    const wrapper = shallow(<Detailcomp name="Ayush Singh" email="tejasvasingh.83@gmail.com" date="2021-07-14" number={9161463059} timing="09:00" interviewer="snehalata" status={false}/>);
    expect(wrapper).toMatchSnapshot();
});

it('checking rendering corectly or not', () => {
    const wrapper = shallow(<Detailcomp name="Ayush Singh" email="tejasvasingh.83@gmail.com" date="2021-07-14" number={9161463059} timing="09:00" interviewer="snehalata" status={false}/>);
    expect(wrapper.length).toEqual(1);
});

it('button clicking or not', () => {
    const mockCallBack = jest.fn();
    const wrapper = shallow(<Detailcomp name="Ayush Singh" email="tejasvasingh.83@gmail.com" date="2021-07-14" number={9161463059} timing="09:00" interviewer="snehalata" status={false}/>);
    wrapper.find('[id="toggleButton"]'.length).tobe(1);
});
