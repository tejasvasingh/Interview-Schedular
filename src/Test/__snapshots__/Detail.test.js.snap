// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creating details compoennet snapshot 1`] = `ShallowWrapper {}`;
